### PR TITLE
feat(app): use webpack-dev-middleware and get/set players into UI

### PR DESCRIPTION
**Pull request by VenceslasProuteau**
@YoannDelpierre

### CODE REVIEW

File: mock/index.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+const express = require('express');
+const app = express();
+const webpack = require('webpack')
+const config = require('../build/webpack.dev.conf')
+const compiler = webpack(config)
+const routes = require('./routes/index');
 
-var routes = require('./routes/index');
+// handle fallback for HTML5 history API
+app.use(require('connect-history-api-fallback')())
```
Comment (YoannDelpierre) [inline mock/index.js]: à mon avis, pas besoin

**Comment by YoannDelpierre** [comment]
Séparer les commits entre la modif sur l'api et le front

File: mock/index.js [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
 
-app.use(routes);
+// serve webpack bundle output
+app.use(require('webpack-dev-middleware')(compiler, {
```
Comment (YoannDelpierre) [inline mock/index.js:9]: needed?

File: src/components/Players.vue [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  export default {
+    name: 'players',
+    components: {},
+    data () {
+      return {
+        players: []
+      }
+    },
+    created () {
+      this.$http.get(ROUTES.PLAYERS)
```
Comment (YoannDelpierre) [inline src/components/Players.vue]: make an external file which wrapp get/then/catch and do not expose $http directly into a file vue

**Comment by vence92** [comment]
@YoannDelpierre

File: src/constants.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export const API_URL = '/api/'
+export const ROUTES = {
+  PLAYERS: API_URL + 'players'
```
Comment (YoannDelpierre) [inline src/constants.js:3]: use interpolation string

**Review by YoannDelpierre — APPROVED** [review]

**Review by YoannDelpierre — APPROVED** [review]

**Pull request merged**